mitosheet/mitosheet/mito_backend.py

```diff
@@ -116,6 +118,28 @@ def __init__(
 
         self.theme = theme
 
+    @property
+    def fully_parameterized_function(self) -> List[str]:
+        """
+        Returns the fully parameterized function string. This is used for
+        cases where we want to get the function string regardless of the 
+        code options the user provided. 
+        """
+        return get_script_as_function(
+            self.steps_manager,
+            [],
+            transpile(self.steps_manager, add_comments=False),
+            self.steps_manager.code_options.get('function_name'),
+            [
+                'import_dataframe',
+                'file_name_export_excel',
+                'file_name_export_csv',
+                'file_name_import_excel',
+                'file_name_import_csv',
+            ],
```

Comment:
My feeling is that we will most likely forget to update this code when we add new parameterization options. Do you wanna add a new valid param called `all` - which just turns all parametrizable params into a param for the function?